### ISSUE TITLE
Source code release?

### ISSUE THREAD

**Issue by eisenbe7** [50ButtonsEach/fliclib-linux-dist#42]
There hasn't been a release of this library since January.

Can you consider releasing the source code and letting us help out?  My flics have been sitting idle for the past few months and I'd really like to use them.  I can't think of many reasons to keep the source closed at this point.

**Comment by Roaders** [comment]
Yes please! I have pretty much given up on the mobile app and have a raspberry pi with node red that I want to use my flics on. Please make this open source so that there is more chance that the issue can get fixed and we can actually use the buttons that we have paid for.

**Comment by fabianbergmark** [comment]
Hi

See the update in the Readme

Cheers, Fabian

**Comment by faceless2** [comment]
Good move. Bluez is a blight on Linux - missing documentation, example scripts pressed into service as system tools, incompatible changes from release 4 to 5 with no clear migration plan... it's a mess, and I'm looking forward to testing your new API.

**Comment by fabianbergmark** [comment]
Hi

See the update in the Readme

Cheers, Fabian

**Comment by faceless2** [comment]
New API integrated with my Java app already and working 100% 👍 Much easier! Thanks very much!

**Comment by eisenbe7** [comment]
Very exciting!

Any chance of a native python implementation of the protocol for the other repo? For the other devices I am using this with (sonos, hue bulbs, etc), python seems to be a popular choice.

**Comment by Emill** [comment]
Yeah Python seems like a good idea. Haven't had time for that.

**Comment by eisenbe7** [comment]
With some initial testing this seems promising.
Can we expect a python implementation on the way? Or will we need to implement that ourselves?

**Comment by Emill** [comment]
It's on its way ;)
Den 10 maj 2016 06:26 skrev "eisenbe7" notifications@github.com:

**Comment by eisenbe7** [comment]
Sweet!

**Comment by Emill** [comment]
Just added a python lib!
